plugins/woocommerce-admin/client/order-attribution/order-attribution-install-banner.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { Button, Card, CardBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import OrderAttributionInstallBannerImage from './order-attribution-install-banner-image';
+import useOrderAttributionInstallBanner from './use-order-attribution-install-banner';
+import './style.scss';
+
+export const OrderAttributionInstallBanner = ( {
+	bannerImage = <OrderAttributionInstallBannerImage />,
+} ) => {
+	const {
+		isOrderAttributionInstallBannerDismissed,
+		dismissOrderAttributionInstallBanner,
+		shouldShowBanner,
+	} = useOrderAttributionInstallBanner();
+
+	if ( ! shouldShowBanner || isOrderAttributionInstallBannerDismissed ) {
+		return null;
+	}
+
+	recordEvent( 'order_attribution_install_banner_viewed' );
```

Comment:
Since bannerImage is a customizable parameter I assume this component will be reusable somewhere else? 

If so, should we add some props for the recordEvents to add information about which of the banners was clicked or viewed?

Reply:
Fixed in

https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-3815cff3f1a08b694ce40b4a0699e0cf64d9f7d0b0f3694313ae78d0bb12f56bR28-R31

https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-3815cff3f1a08b694ce40b4a0699e0cf64d9f7d0b0f3694313ae78d0bb12f56bL80-R83

https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-09863637c86d1cdb04cfa551d822f0ce78dd4589e68783b05e753d90b956da6fL24-R28